Apps/AdminWebClient/src/Server/Services/CommunicationService.cs

```diff
@@ -49,6 +49,13 @@ public CommunicationService(ILogger<CommunicationService> logger, ICommunication
         /// <inheritdoc />
         public RequestResult<Communication> Add(Communication communication)
         {
+            if (/* is email */ false) {
```

Comment:
any reasons that we don't want to remove this block of code completely ?

Reply:
weird, that code is outdated. can you refresh? that code shouldn't be there anymore